### ISSUE TITLE
Running?

### ISSUE THREAD

**Issue by rboarman** [danielcbright/cross-account-cloudwatch-logs-to-kinesis#1]
Did you ever get this working? I'm struggling with the same thing... Thanks!

**Comment by ArisBee** [comment]
Yes the same, it's in AWS documentation since 2015 [here](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/cwl-ug.pdf). But no way to receive logs in the _testDestination_ group although I don't get any error.
Besides, to do it you have to use a root API key which is contrary to AWS security best practices, I contacted them about this issue and they answered they will fix it later...
